stream_chat.go

```diff
@@ -49,6 +49,7 @@ type StreamChannel interface {
 	GetReplies(parentID string, options map[string][]string) (replies []*Message, err error)
 	MarkRead(userID string, options map[string]interface{}) error
 	RemoveMembers(userIDs ...string) error
+	Reload() error
```

Comment:
This is a temporary workaround for the lacking synchronization facilities. Document that.

Reply:
I'm not sure if it's temporary and will server-side SDK ever have event listeners :-D

Reply:
Yes, I'm working hard on that (with a full-blown event stream). Should be Q4.